http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmDao.scala

```diff
@@ -333,7 +334,7 @@ object WsmDecoders {
     Decoder.forProduct1("landingzones")(ListLandingZonesResult.apply)
 
   implicit val landingZoneResourceDecoder: Decoder[LandingZoneResource] =
-    Decoder.forProduct5("resourceId", "resourceType", "resourceName", "resourceParentId", "region")(
+    Decoder.forProduct6("resourceId", "resourceType", "resourceName", "resourceParentId", "region", "tags")(
```

Comment:
Out of scope for this PR, just a comment. It occurs to me that for newer things we've started using the published WSM Jave client in Leo, and deprecating `WsmDao/HttpWsmDao`. Perhaps Leo can also start using the LZ Service Java client in the future.

Reply:
I had the same thought, it felt strange to be de/serializing this stuff manually